go.mod

```diff
@@ -4,6 +4,6 @@ go 1.15
 
 require (
 	github.com/edsrzf/mmap-go v1.1.0
-	go.mozilla.org/pkcs7 v0.0.0-20210826202110-33d05740a352
+	github.com/secDre4mer/pkcs7 v0.0.0-20240322103146-665324a4461d
```

Comment:
I was hoping we apply the patches you made over: https://github.com/smallstep/pkcs7

Because @smallstep took a couple of PRs from the mozilla fork and applied them to his fork. If it makes sense, you can submit a PR on his repo.

 If you think it is not relevant for people to benefit from the commits you did, then we can keep it this way.

Reply:
This sounds good. I noticed that the mozilla repo was archived and no longer maintained and therefore created my own branch, but if there's a supported successor, I'll gladly create a PR there.

Reply:
The smallstep/pkcs7 maintainer hasn't responded to the PR for some weeks now... How should we proceed here?

Reply:
Thank for your efforts @secDre4mer, let's merge this one then.